Source/ACE.Server/WorldObjects/WorldObject_Properties.cs

```diff
@@ -2409,12 +2409,39 @@ public string QuestRestriction
         /* pressure plates */
 
         /// <summary>
-        /// Returns TRUE if this object can be activated (default)
+        /// Returns Active.Active or Active.SecondaryAction, if this object can be activated (1 is default)
+        /// 23 Weenies in Cache.bin contain Active 65535 or 0xffff
         /// </summary>
-        public bool Active
+        private Active Active
```

Comment:
there is no need for this until there is actually logic defined based on the difference between Active=1 and Active=0xFFFF